src/pages/ListScholarships.jsx

```diff
@@ -110,10 +112,13 @@ function ListScholarships() {
         <title>Search Scholarships</title>
       </Helmet>
       <Typography variant="h3" component="h1" align="center">
-        Scholarships
+        {t('listScholarships.title')}
       </Typography>
       <FilterBar queryParams={params} {...{ setQueryParam }} />
```

Comment:
The filter bar is missing translations

Reply:
Should we also translate all the grade and major options?

Reply:
Yeah that would be good. Though I guess a separate PR is fine.